content/pandas/concepts/dataframe/terms/at/at.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.at[]'
+Description: 'Access a specific value in row/column pairs in a dataset.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+In Pandas, the **`.at[]`** function is used as an accessor to fetch a specific value located in a DataFrame with the aid of row/column pairs. 
+
+## Syntax
+
+```pseudo
+dataframe.at[position, label]
+```
+-`position`:The label of the row to be accessed.
+-`label`:Name of the column to be accessed.
+
+The result returned is a single element located at the specified position within the DataFrame.
+
+## Example
+
+The following example shows the use of the `.at[]` accessor function:
+
+```py
+import pandas as pd
+
+df = pd.DataFrame({'A': [1, 2], 'B': [3, 4]})
+element = df.at[0, 'B']
+
+print(element)  # Output: 3
```

Comment:
```suggestion
import pandas as pd

# Create a DataFrame with two columns 'A' and 'B'
df = pd.DataFrame({'A': [1, 2], 'B': [3, 4]})

# Use .at[] to access the element at row 0, column 'B'
element = df.at[0, 'B']

# Print the accessed element
print(element)  
```

Reply:
Added comments for better understanding. Note that you can add comments in the example section, avoid having comments for the codebyte section.